view/next-project/src/components/sponsoractivities/OpenAddModalButton.tsx

```diff
@@ -18,6 +18,7 @@ export default function OpenModalButton(props: Props) {
   return (
     <>
       <AddButton
+        className='fixed bottom-4 right-4 z-50 md:static md:bottom-auto md:right-auto md:z-auto'
```

Comment:
```suggestion
        className='fixed bottom-4 right-4 md:static md:bottom-auto md:right-auto md:z-auto'
```
グローバルナビとかぶってたからz-50消して欲しいー

<img width="431" alt="スクリーンショット 2024-05-10 15 46 56" src="https://github.com/NUTFes/FinanSu/assets/115447919/e93a231b-69cc-416c-8876-41c8f52f2539">